cmd/lakectl/cmd/docs.go

```diff
@@ -52,9 +52,48 @@ This will setup a ` + "`$HOME/.lakectl.yaml`" + ` file with the credentials and
 When setting up a new installation and creating initial credentials (see [Quickstart](../quickstart/index.md)), the UI
 will provide a link to download a preconfigured configuration file for you.
 
-` + "`lakectl`" + ` configuration items can each be controlled by an environment variable. The variable name will have a prefix of
-*LAKECTL_*, followed by the name of the configuration, replacing every '.' with a '_'. Example: ` + "`LAKECTL_SERVER_ENDPOINT_URL`" + ` 
-controls ` + "`server.endpoint_url`" + `.
+## lakectl Configuration
+
+` + "`lakectl`" + ` reads its configuration from a YAML file (default path ` + "`~/.lakectl.yaml`" + `, overridable with ` + "`--config`" + `) and/or from environment variables.
+
+* Every configuration key can be supplied through an environment variable using the pattern ` + "`LAKECTL_<UPPERCASE_KEY_WITH_DOTS_REPLACED_BY_UNDERSCORES>`" + `.
+* Any value given on the command-line flags overrides the value in the configuration file, which in turn overrides the value supplied through the environment.
+
+### Reference
+
+* ` + "`credentials.access_key_id` `(string : required)`" + ` – Access-key ID used to authenticate against lakeFS.
+* ` + "`credentials.secret_access_key` `(string : required)`" + `  – Secret access key paired with the access key ID.
+* ` + "`credentials.provider.type` `(string : \"\"" + ")`" + ` – Set to ` + "`aws_iam`" + ` to obtain temporary credentials from AWS IAM; empty for static credentials (default).
```

Comment:
consider comment that this requires server side support which is part of enterprise

Reply:
Done.